nuxeo-quota-core/src/test/java/org/nuxeo/ecm/quota/count/TestDocumentsSizeUpdater.java

```diff
@@ -1264,62 +1264,59 @@ protected void doUpdateAndVersionContent() throws Exception {
         next();
     }
 
-    protected void doSimpleVersion() throws Exception {
+    protected void doSimpleVersion() {
         DocumentModel firstFile = session.getDocument(firstFileRef);
         firstFile.setPropertyValue("dc:title", "a version");
         firstFile.putContextData(VersioningService.VERSIONING_OPTION, VersioningOption.MINOR);
         firstFile = session.saveDocument(firstFile);
         next();
     }
 
-    protected void doRemoveFirstVersion() throws Exception {
+    protected void doRemoveFirstVersion() {
         List<DocumentModel> versions = session.getVersions(firstFileRef);
         session.removeDocument(versions.get(0).getRef());
         next();
     }
 
-    protected void doRemoveContent() throws Exception {
+    protected void doRemoveContent() {
         session.removeDocument(firstFileRef);
         next();
     }
 
-    protected void doRemoveFolderishContent() throws Exception {
+    protected void doRemoveFolderishContent() {
         session.removeDocument(firstSubFolderRef);
         next();
     }
 
-    protected void doDeleteFileContent() throws Exception {
-        List<DocumentModel> docs = new ArrayList<DocumentModel>();
+    protected void doDeleteFileContent() {
+        List<DocumentModel> docs = new ArrayList<>();
         docs.add(session.getDocument(firstFileRef));
         Framework.getService(TrashService.class).trashDocuments(docs);
         next();
     }
 
-    protected void doDeleteFileContent(final DocumentRef fileRef) throws Exception {
-        List<DocumentModel> docs = new ArrayList<DocumentModel>();
-        docs.add(session.getDocument(fileRef));
-        Framework.getService(TrashService.class).trashDocuments(docs);
+    protected void doDeleteFileContent(final DocumentRef fileRef) {
+        DocumentModel doc = session.getDocument(fileRef);
+        Framework.getService(TrashService.class).trashDocument(doc);
         next();
     }
 
-    protected void doUndeleteFileContent() throws Exception {
-        List<DocumentModel> docs = new ArrayList<DocumentModel>();
-        docs.add(session.getDocument(firstFileRef));
-        Framework.getService(TrashService.class).undeleteDocuments(docs);
+    protected void doUndeleteFileContent() {
+        Framework.getService(TrashService.class).untrashDocument(session.getDocument(firstFileRef));
         next();
     }
 
-    protected void doCopyContent() throws Exception {
+    protected void doCopyContent() {
         session.copy(firstFileRef, secondSubFolderRef, null);
         next();
     }
 
-    protected void doCopyFolderishContent() throws Exception {
+    protected void doCopyFolderishContent() {
         session.copy(firstSubFolderRef, secondFolderRef, null);
         next();
     }
 
-    protected void dump() throws Exception {
+    protected void dump() {
```

Comment:
Again all this cleanup is nice but should be in a separate commit, for readability.